City-of-Helsinki/berth-reservations-admin
# VEN-526 | Update styles and fix missing translations for Harbor details

## Description :sparkles:

* Use Icon in ExternalLink arrow
* Update styles and fix missing translations for Harbor details
* Update styles and add harbor map links for IndividualHarborPage

## Issues :bug:

### Closes :no_good_woman:

* [VEN-526](https://helsinkisolutionoffice.atlassian.net/browse/VEN-526): Frontend: update styles and fix missing translations for Harbor details

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

* Updated tests for HarborDetails with with new data

### Manual testing :construction_worker_man:

1. Check the appearance of harbor details
2. Check the appearance of individual harbor page

## Screenshots :camera_flash:

## Additional notes :spiral_notepad:
